Gemfile

```diff
@@ -28,9 +28,27 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'rb-readline' #make pry work
+gem 'figaro' #hide our stuff
+gem 'serializer' #api serializers
+gem 'faraday' #for http requests
+
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
+  gem 'pry' #duh
+  gem 'launchy' #because restarting the server every time sucks
+  gem 'faker' # for seed data
+
+  gem 'rspec-rails' #testing suite
+  gem 'capybara' #feature testing
+  gem 'factory_girl_rails' #creating items for tests
+  gem 'simplecov' #so we can tell what's being hit
+  gem 'database_cleaner' #keep it clean
+  gem 'vcr' #for testing our api's
```

Comment:
@ vido Awesome thanks for adding these! As a best practice, `vcr` and `webmock`  should be in a `test` only group since they are only used in the test environment.